Run the tests in a fixed order.

These changes will be passed upstream.

--- serf-1.3.10/build/check.py.orig	2024-07-24 22:14:16.260039964 +0200
+++ serf-1.3.10/build/check.py	2024-07-24 22:17:36.976438254 +0200
@@ -51,7 +51,7 @@
   TEST_ALL_EXE = os.path.join(test_builddir, TEST_ALL_EXE)
 
   # Find test responses and run them one by one
-  for case in glob.glob(testdir + "/testcases/*.response"):
+  for case in sorted(glob.glob(testdir + "/testcases/*.response")):
     print("== Testing %s ==" % (case))
     try:
       subprocess.check_call([SERF_RESPONSE_EXE, case])
